CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/resourcetypes/CVKCommandBuffer.java

```diff
@@ -139,282 +146,8 @@ public void FinishRecord(){
-//	}
-//	
-//	public void drawCmd(int vertexCount){
-//		
-//		vkCmdDraw(vkCommandBuffer, vertexCount, 1, 0, 0);
-//	}
-//	
-//	public void dispatchCmd(int groupCountX, int groupCountY, int groupCountZ){
-//		
-//		vkCmdDispatch(vkCommandBuffer, groupCountX, groupCountY, groupCountZ);
-//	}
-//	
-//	public void copyBufferCmd(long srcBuffer, long dstBuffer,
-//								    long srcOffset, long dstOffset,
-//								    long size){
-//		
-//		VkBufferCopy.Buffer copyRegion = VkBufferCopy.calloc(1)
-//					.srcOffset(srcOffset)
-//					.dstOffset(dstOffset)
-//					.size(size);
-//		
-//		vkCmdCopyBuffer(vkCommandBuffer, srcBuffer, dstBuffer, copyRegion);
-//	}
-//	
-//	public void copyBufferToImageCmd(long srcBuffer, long dstImage, int width, int height, int depth){
-//		
-//		VkBufferImageCopy.Buffer copyRegion = VkBufferImageCopy.calloc(1)
-//					.bufferOffset(0)
-//					.bufferRowLength(0)
-//					.bufferImageHeight(0);
-//		
-//		VkImageSubresourceLayers subresource = VkImageSubresourceLayers.calloc()
-//					.aspectMask(VK_IMAGE_ASPECT_COLOR_BIT)
-//					.mipLevel(0)
-//					.baseArrayLayer(0)
-//					.layerCount(1);
-//		
-//		VkExtent3D extent = VkExtent3D.calloc()
-//					.width(width)
-//					.height(height)
-//					.depth(depth);
-//		
-//		VkOffset3D offset = VkOffset3D.calloc()
-//					.x(0)
-//					.y(0)
-//					.z(0);
-//		
-//		copyRegion.imageSubresource(subresource);
-//		copyRegion.imageExtent(extent);
-//		copyRegion.imageOffset(offset);
-//	
-//		vkCmdCopyBufferToImage(vkCommandBuffer, srcBuffer, dstImage,
-//				VK_IMAGE_LAYOUT_TRANSFER_DST_OPTIMAL, copyRegion);
-//	}
-//	
-//	public void pipelineImageMemoryBarrierCmd(long image, int oldLayout, int newLayout,
-//			int srcAccessMask, int dstAccessMask, int srcStageMask, int dstStageMask,
-//			int baseMipLevel, int mipLevelCount){
-//		
-//		VkImageMemoryBarrier.Buffer barrier = VkImageMemoryBarrier.calloc(1)
-//				.sType(VK_STRUCTURE_TYPE_IMAGE_MEMORY_BARRIER)
-//				.oldLayout(oldLayout)
-//				.newLayout(newLayout)
-//				.srcQueueFamilyIndex(VK_QUEUE_FAMILY_IGNORED)
-//				.dstQueueFamilyIndex(VK_QUEUE_FAMILY_IGNORED)
-//				.image(image)
-//				.srcAccessMask(srcAccessMask)
-//				.dstAccessMask(dstAccessMask);
-//		
-//		barrier.subresourceRange()
-//				.aspectMask(VK_IMAGE_ASPECT_COLOR_BIT)
-//				.baseMipLevel(baseMipLevel)
-//				.levelCount(mipLevelCount)
-//				.baseArrayLayer(0)
-//				.layerCount(1);
-//	
-//		vkCmdPipelineBarrier(vkCommandBuffer, srcStageMask, dstStageMask,
-//				VK_DEPENDENCY_BY_REGION_BIT, null, null, barrier);
-//		
-//		barrier.free();
-//	}
-//	
-//	public void pipelineImageMemoryBarrierCmd(long image, int srcStageMask, int dstStageMask,
-//			VkImageMemoryBarrier.Buffer barrier){
-//	
-//		vkCmdPipelineBarrier(vkCommandBuffer, srcStageMask, dstStageMask,
-//				VK_DEPENDENCY_BY_REGION_BIT, null, null, barrier);
-//	}
-//	
-//	public void pipelineMemoryBarrierCmd(int srcAccessMask, int dstAccessMask,
```

Comment:
I've actually added a lot of these back in to reduce the code in the renderables RecordCmdBuffer functions.  It might be best to leave the commented code back in for this change as I'm working on the class

Reply:
I can't know what you have or haven't added.  It should be easy enough for you to merge though.